let s = sum(1, 2)
func sum(_ a: Int, _ b: Int) -> Int {
    a + b
}
/*
func isTwo(_ a: Int) -> String {
    if a == 2 {
        return "Это двойка"
    } else {
        return "Это не двойка"
    }
}
let a = isTwo(3)
*/
/*
func sum() -> Int {
    print("Раз")
    return 10
    print("Два")
}
 
let a = sum()
 */
/* В консоли мы увидим
 Раз
 Program ended with exit code: 0
 */
/*
func getUserFullName() -> String {
    print("Введите ваше имя:")
    let name = readLine() ?? ""
    print("Выедите вашу фамилию:")
    let surname = readLine() ?? ""
    return name + "" + surname
}
let userFullName = getUserFullName()
print(userFullName)
 */
/*
 func sum(_ a: Int, _ b: Int) {
     print(a + b)
 }
 func sum(_ a: Int, _ b: Int) -> Int {
     return a + b
 }
 func sum(_ a: Double, _ b: Double) {
     print(a + b)
 }
 func sum(first a: Int, _ b: Int) {
     print(a + b)
 }
 */
/*
func sum(_ a: Int, _ b: Int) {
    print(a + b)
}
sum(1, 1)
*/
/*
func sum(_ a: Int, _ b: Int) -> Void {
    print(a + b)
}
sum(1, 1)
*/
/*
func sum(_ a: Int, _ b: Int) -> String {
    return String(a + b)
}
let a: String = sum(1, 1)
print(a)
*/
/*
func sum(_ a: Int, _ b: Int) -> Int {
    return a + b
}
let a: Int = sum(1, 1)
print(a)
*/
/*
func sum(
    _ first: Int,
    second: Int,
    thridNumber thrid: Int
) -> Int {
    return first + second + thrid
}
let a = sum(1, second: 2, thridNumber: 3) // 6
 */
/*
 func sum(
     _ first: Int,
     _ second: Int,
     _ thrid: Int
 ) -> Int {
     return first + second + thrid
 }
 let a = sum(1, 2, 3) // 6
 */
/*
 func sum(
     first: Int,
     second: Int,
     thrid: Int
 ) -> Int {
     return first + second + thrid
 }
 let a = sum(first: 1, second: 2, thrid: 3) // 6
 */
/*
func sum(
    firstNumber first: Int,
    secondNumber second: Int,
    thridNumber thrid: Int
) -> Int {
    return first + second + thrid
}
*/
/*
let a = sum(firstNumber: 4, secondNumber: 3, thridNumber: 2) // 9
 */
/*
func sum() -> Int {
    return 5 + 5
}
let a = sum() // 10
let b = sum() // 10
 */
/*
func sum(firstNumber first: Int, secondNumber second: Int) -> Int {
    return first + second
}
let a = sum(firstNumber: 5, secondNumber: 5)
let b = sum(firstNumber: 2, secondNumber: 9)
print(a, b)
 */
/* В консоле мы увидим
 10 11
 Program ended with exit code: 0
 
*/
